client.go

```diff
@@ -245,6 +245,10 @@ func request(ctx context.Context, accessToken string, method string, url string,
 		if err = checkErrorResponse(body); err != nil {
 			return nil, ConstructNestedError("request returned an error", err)
 		}
+		if resp.StatusCode == 500 {
+			// this is a database error
+			return nil, fmt.Errorf("%s", string(body))
+		}
 		return nil, fmt.Errorf("request returned non ok status code: %d", resp.StatusCode)
```

Comment:
Does it make sense to print error body for other status codes as well?

Reply:
Hm, makes sense, will add it as well